gapis/resolve/command_tree.go

```diff
@@ -91,6 +91,7 @@ func CommandTreeNode(ctx context.Context, c *path.CommandTreeNode) (*service.Com
 			NumChildren: item.Count(),
 			Commands:    cmdTree.path.Capture.CommandRange(uint64(item.Range.First()), uint64(item.Range.Last())),
 			Group:       item.Name,
+			NumCommands: item.DeepCount(func(g atom.Group) bool { return true; /* TODO: Subcommands */ }),
```

Comment:
Nit: I'm a little bit concerned with the number of deep traversals required to calculate counts. Seems like this could be calculated at `AddGroup` / `AddAtoms` time. That said, I'm happy to ignore this until it shows up in a profile.